### ISSUE TITLE
Error happened

### ISSUE THREAD

**Issue by LetaiefAymen** [avocode/avocode#1498]
IMPORTANT: Please try to describe how did it happen.
i clicked on the "Open in Avocode App" from the website (would be nice to say that it's already open or something, because i thought it's not working but it was) then i clicked on the screen in avocode
-----
App v. `2.14.4`
Avocode UID: 54517
```
TypeError: Cannot read property 'get' of undefined
    at e.getLayerIdsToSelect (file:///Users/aymen/Downloads/Avocode-darwin-x64/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:2662108)
    at file:///Users/aymen/Downloads/Avocode-darwin-x64/Avocode.app/Contents/Resources/app.asar/dist/avocode.min.js:1:2659302
    at invokeFunc (/Users/aymen/Downloads/Avocode-darwin-x64/Avocode.app/Contents/Resources/app.asar/node_modules/lodash/lodash.js:10350:23)
    at trailingEdge (/Users/aymen/Downloads/Avocode-darwin-x64/Avocode.app/Contents/Resources/app.asar/node_modules/lodash/lodash.js:10397:18)
    at timerExpired (/Users/aymen/Downloads/Avocode-darwin-x64/Avocode.app/Contents/Resources/app.asar/node_modules/lodash/lodash.js:10385:18)
    at wrapped (/Users/aymen/Downloads/Avocode-darwin-x64/Avocode.app/Contents/Resources/app.asar/node_modules/raven-js/src/raven.js:314:29)
```

**Comment by vojtatranta** [comment]
That's a bug, we'll fix it!

**Comment by jakubzitny** [comment]
We've just released `2.14.5` with a fix for this bug, please update the app and let us know if you have any more problems 😉
